docs/PacketFence_Network_Devices_Configuration_Guide.asciidoc

```diff
@@ -2167,6 +2167,86 @@ interface GigabitEthernet0/0/8
 
 ----
 
+IBM
+~~
+
+RackSwitch G8052
+^^^^^^^^^^^^^^^^
+
+PacketFence supports only the 802.1x. It's tested and developped for the Firmware 7.9.11.0.
```

Comment:
*supports only 802.1x authentication.

*It has been tested on <NAME OF IBM OS HERE> version 7.9.11.0